openedx/core/djangoapps/user_api/accounts/forms.py

```diff
@@ -0,0 +1,40 @@
+"""
+Django forms for accounts
+"""
+
+from django import forms
+from django.core.exceptions import ValidationError
+
+
+class RetirementQueueDeletionForm(forms.Form):
+    """
+    Admin form to facilitate learner retirement cancellation
+    """
+    cancel_retirement = forms.BooleanField(required=True)
```

Comment:
Should this be checked at all in the save function?

Reply:
or, does required=True make it so that it needs to be clicked to pass validation?

Reply:
Yeah the required attribute allows Django to handle the validation for us.